test/Rings/mpoly-localizations.jl

```diff
@@ -342,14 +344,14 @@ end
   @test is_zero_divisor(Z4(2))
   @test is_zero_divisor(Z4x(2))
   @test is_zero_divisor(f)
+  L, _ = localization(Z4x, Z4x(x))
+  @test !is_zero_divisor(L(5))
+  @test is_zero_divisor(L(2))
+  @test !is_zero_divisor(L(x))
+  @test is_zero_divisor(2*L(x))
+  @test is_zero_divisor(zero(L))
   # At the moment, the Singular backend complains when the modulus of the coefficient 
   # ring is not a prime. So we leave the following tests out for the time being. 
-#  L, _ = localization(Z4x, Z4x(x))
-#  @test !is_zero_divisor(L(5))
-#  @test is_zero_divisor(L(2))
-#  @test !is_zero_divisor(L(x))
-#  @test is_zero_divisor(2*L(x))
-#  @test is_zero_divisor(zero(L))
```

Comment:
Why remove these tests? They should run one day, eventually. Once somebody gets around to implement the necessary routines, they might want to know what to measure them against.

Reply:
They were not removed, but the comment moved down and the '#' were removed, because these tests work now!

Reply:
These tests moved some lines further up above the comment, because they turned out to work by now, and the '#' has been removed to make sure that they are run upon testing from now on. (Compare the red and the green part in the diff of this file -- these are identical tests)